WooCommerce/UITestsFoundation/Screens/Settings/SettingsScreen.swift

```diff
@@ -60,19 +51,3 @@ public final class SettingsScreen: ScreenObject {
         return try BetaFeaturesScreen()
     }
 }
-
-/// Assertions
-extension SettingsScreen {
-
-    public func verifySelectedStoreDisplays(storeName expectedStoreName: String, siteUrl expectedSiteUrl: String) -> SettingsScreen {
-        let actualStoreName = selectedStoreNameGetter(app).label
-        let expectedSiteUrl = expectedSiteUrl.replacingOccurrences(of: "http://", with: "")
-        let actualSiteUrl = selectedSiteUrlGetter(app).label
-
-        XCTAssertEqual(expectedStoreName, actualStoreName,
-                       "Expected display name '\(expectedStoreName)' but '\(actualStoreName)' was displayed instead.")
-        XCTAssertEqual(expectedSiteUrl, actualSiteUrl,
-                       "Expected site URL \(expectedSiteUrl) but \(actualSiteUrl) was displayed instead.")
-        return self
-    }
-}
```

Comment:
This is no longer displayed on Settings but is on the Hub Menu instead, this is moved to MenuScreen.